recipes/commlib-py/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "commlib-py" %}
+{% set version = "0.10.6" %}
+{% set commit = "b812d91bdad45ceab84003d372683e6fd6985719" %}
+{% set sha256 = "a2ead2622c29481d1cb1382f51bb28ee0963f01df0579d71071f2bd926deface" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/robotics-4-all/commlib-py/archive/{{ commit }}.tar.gz
```

Comment:
Please use the release tag archive.

Reply:
The owner of the package seems to have discontinued the github releases. I will then revert this recipe to using the pypi package, add the licence and test in the recipe, will this work better?

Reply:
Reverted to using the more recent pypi package